docs/owo/setup.md

```diff
@@ -34,13 +34,34 @@ Then, declare the dependency inside your `dependencies` block and as well as the
 === "build.gradle (Neoforge)"
     ```groovy 
     dependencies {
-        // Moddev Projects - Neoforge
+        // Vanilla/Moddev Projects - Neoforge
         implementation "io.wispforest:owo-lib-neoforge:${project.owo_version}"
         accessTransformer "io.wispforest:owo-lib-neoforge:${project.owo_version}"
         interfaceInjectionData "io.wispforest:owo-lib-neoforge:${project.owo_version}"
 
         // Arch Loom Projects - Neoforge
         modImplementation "io.wispforest:owo-lib-neoforge:${project.owo_version}"
+
+        // Required due to issues with Arch Loom and JIJ within neo. May require bumping the version every once and awhile.
```

Comment:
Is the case still that Arch Loom requires manually including these?

Reply:
Yes as I attempted to ask about it but not much discussion about such took place and overall development seems kinda staled/inactive